app/controllers/insights_cloud/hits_controller.rb

```diff
@@ -22,6 +23,14 @@ def model_of_controller
       ::InsightsHit
     end
 
+    def save_setting_and_sync
+      token = Setting::RhCloud.find_by_name("rh_cloud_token")
+      token.value = params[:value]
+      token.save
```

Comment:
Let's use the version that throws exception on validation.
```suggestion
      token.save!
```